src/server.rs

```diff
@@ -0,0 +1,294 @@
+use std::env;
+use fresh_eyes::{
+    extract_pr_details, get_pull_request_reviews, Branch as LibBranch,
+    ForkRequest as LibForkRequest, PullRequest as LibPullRequest,
+};
+use fresheyes::git_hub_service_server::{GitHubService, GitHubServiceServer};
+use fresheyes::{Branch, ForkRequest, ForkResult, PrResponse, PullRequest, PullRequestDetails};
+use tonic::{transport::Server, Request, Response, Status};
+use dotenv::dotenv;
+
+pub mod fresheyes {
+    tonic::include_proto!("fresheyes");
+}
+
+#[derive(Debug, Default)]
+pub struct GitHubServiceImpl {
+    github_token: String,
```

Comment:
```suggestion
    mod github_token: String,
```

Reply:
Should be pub not mod. Made a mistake the 1st time

Reply:
yea, made the change